src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
+  import type { QuizData } from '$lib/constants/quiz';
+  import LeftArrow from '../icons/leftArrow.svelte';
+  import RightArrow from '../icons/rightArrow.svelte';
+  import MoreInfo from './moreInfo.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
+  let response: string[] = [];
+  let noMoreQuestions: boolean = false;
+  let answeredEveryQuestion: boolean = false;
```

Comment:
Once the user has answered every question, a third button should appear that leads to the results page.
We might be able to use a reactive statement to omit the checks in your `recordAnswer` function.
```suggestion
  $: answeredAllQuestions = responses.length === data.questions.length && !responses.includes(undefined);
```

Reply:
Done